wxJoystick stubs don't compile

Error on platform without wxJoystick:
```
In file included from ../../../../sip/cpp/sip_advwxWizardPage.cpp:10:
../../../../sip/cpp/sipAPI_adv.h:3049:21: error: expected ‘;’ at end of member declaration
         { return 0; }
                     ^
                      ;
../../../../sip/cpp/sipAPI_adv.h:3052:21: error: expected ‘;’ at end of member declaration
         { return 0; }
                     ^
                      ;
../../../../sip/cpp/sipAPI_adv.h:3049:11: error: expected primary-expression before ‘return’
         { return 0; }
           ^~~~~~
../../../../sip/cpp/sipAPI_adv.h:3049:11: error: expected ‘}’ before ‘return’
../../../../sip/cpp/sipAPI_adv.h:3049:11: error: cannot convert ‘<brace-enclosed initializer list>’ to ‘int’ in initialization
../../../../sip/cpp/sipAPI_adv.h:3049:11: error: expected ‘;’ before ‘return’
../../../../sip/cpp/sipAPI_adv.h:3052:11: error: expected primary-expression before ‘return’
         { return 0; }
           ^~~~~~
../../../../sip/cpp/sipAPI_adv.h:3052:11: error: expected ‘}’ before ‘return’
../../../../sip/cpp/sipAPI_adv.h:3052:11: error: cannot convert ‘<brace-enclosed initializer list>’ to ‘int’ in initialization
../../../../sip/cpp/sipAPI_adv.h:3052:11: error: expected ‘;’ before ‘return’
```
Which corresponds to:
```
    int GetMaxButtons
        { return 0; }

    int GetMaxAxes
        { return 0; }

```
Looks like these functions are missing their `()`.

--- a/etg/joystick.py
+++ b/etg/joystick.py
@@ -36,8 +36,8 @@ def run():
     assert isinstance(c, etgtools.ClassDef)
     c.mustHaveApp()
 
-    c.addItem(MethodDef(name='GetMaxButtons', type='int', isConst=True))
-    c.addItem(MethodDef(name='GetMaxAxes', type='int', isConst=True))
+    c.addItem(MethodDef(name='GetMaxButtons', type='int', isConst=True, argsString='() const'))
+    c.addItem(MethodDef(name='GetMaxAxes', type='int', isConst=True, argsString='() const'))
 
     tools.generateStubs('wxUSE_JOYSTICK', module)
 

